libs/blocks/global-navigation/utilities/menu/menu.js

```diff
@@ -303,11 +319,12 @@ const decorateMenu = (config) => logErrorFor(async () => {
     const content = await res.text();
     const parsedContent = await replaceText(content, getFedsPlaceholderConfig(), undefined, 'feds');
     const menuContent = toFragment`<div class="feds-menu-content">${parsedContent}</div>`;
-    menuTemplate = toFragment`<div class="feds-popup">
+    menuTemplate = toFragment`<div class="feds-popup" id="feds-popup">
```

Comment:
Since you add the id for styling purposes, you probably can remove this again if you find a different solution.

Reply:
Done 👍